utils/semverutils/semverutils.go

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2024 Renesas Electronics Corporation.
+// Copyright (C) 2024 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package launcher provides set of API to controls services lifecycle
+
+package semverutils
```

Comment:
Should be part of aos_common/utils

Reply:
no objections at all
Anyway, I borrowed this util from Mykola Kobets PR https://github.com/aosedge/aos_communicationmanager/pull/129/files

So once that PR is merged, I will simply remove this WIP commit